core/api/src/app/authentication/login.ts

```diff
@@ -85,7 +85,6 @@ export const loginWithPhoneToken = async ({
   if (validCode instanceof Error) return validCode
 
   await rewardFailedLoginAttemptPerIpLimits(ip)
-  await rewardFailedLoginAttemptPerLoginIdentifierLimits(phone)
```

Comment:
we stop rewarding login, such that more than X login in a timeframe will lead to login error